webhooks/src/main/java/org/openmbee/sdvc/webhooks/controllers/WebhooksController.java

```diff
@@ -0,0 +1,200 @@
+package org.openmbee.sdvc.webhooks.controllers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.openmbee.sdvc.core.config.Privileges;
+import org.openmbee.sdvc.core.dao.ProjectDAO;
+import org.openmbee.sdvc.core.dao.WebhookDAO;
+import org.openmbee.sdvc.core.exceptions.BadRequestException;
+import org.openmbee.sdvc.core.exceptions.NotFoundException;
+import org.openmbee.sdvc.core.objects.BaseResponse;
+import org.openmbee.sdvc.core.security.MethodSecurityService;
+import org.openmbee.sdvc.data.domains.global.Project;
+import org.openmbee.sdvc.data.domains.global.Webhook;
+import org.openmbee.sdvc.webhooks.json.WebhookJson;
+import org.openmbee.sdvc.webhooks.objects.WebhookRequest;
+import org.openmbee.sdvc.webhooks.objects.WebhookResponse;
+import org.openmbee.sdvc.webhooks.services.WebhookEventService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/project")
+public class WebhooksController {
+
+    private WebhookDAO webhookRepository;
+    private ProjectDAO projectRepository;
+    private WebhookEventService webhookEventService;
+    private MethodSecurityService mss;
+    private ObjectMapper om;
+
+    @Autowired
+    public WebhooksController(WebhookDAO webhookRepository, WebhookEventService webhookEventService) {
+        this.webhookRepository = webhookRepository;
+        this.webhookEventService = webhookEventService;
+    }
+
+    @Autowired
+    public void setWebhookRepository(WebhookDAO webhookRepository) {
+        this.webhookRepository = webhookRepository;
+    }
+
+    @Autowired
+    public void setProjectRepository(ProjectDAO projectRepository) {
+        this.projectRepository = projectRepository;
+    }
+
+    @Autowired
+    public void setWebhookEventService(WebhookEventService webhookEventService) {
+        this.webhookEventService = webhookEventService;
+    }
+
+    @Autowired
+    public void setMss(MethodSecurityService mss) {
+        this.mss = mss;
+    }
+
+    @Autowired
+    public void setObjectMapper(ObjectMapper om) {
+        this.om = om;
+    }
+
+    @GetMapping(value = {"", "/{projectId}/webhooks"})
```

Comment:
no need for "" since that'll be projects,
also preauth can take out check for null